translator/trace/zipkin/zipkinv1_thrift_to_protospan_test.go

```diff
@@ -17,6 +17,7 @@ package zipkin
 import (
 	"encoding/binary"
 	"encoding/json"
+	"github.com/google/go-cmp/cmp"
```

Comment:
I pulled in go-cmp because it outputs really nice diffs unlike reflect.DeepEqual

Reply:
Use `assert.EqualValues` as we do in the rest of the repo.

Reply:
Updated for consistency. go-cmp does produce better outputs imo though